### PR TITLE
Rachana_Assignment_18_90min_Done

### PR DESCRIPTION
**Pull request by RachanaGJun20**
prg1
Middlechar
prg3
prg4

### CODE REVIEW

**Review by KrishnaKTechnocredits — APPROVED** [review]
Very good job. Code looks good to me.

File: src/rachanaGJun20/String/StringReplaceFun.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+package rachanaGJun20.String;
+
+public class StringReplaceFun {
+
+	// method will print special char count in a given string
+	void countSpecialChar(String givenstr) {
+		
+
+		String newstr = givenstr;
+		newstr = newstr.replace("Kr", "");
```
Comment (KrishnaKTechnocredits) [inline src/rachanaGJun20/String/StringReplaceFun.java:15]: We should not write it like this. This is completely hardcoded... Your code wont work if name is maulik with same special character. and givenstr has no meaning then why to pass parameter if you want to say replace Kr with "" and so on...

Comment (RachanaGJun20) [reply src/rachanaGJun20/String/StringReplaceFun.java:15]: > Very good job. Code looks good to me.

Thanks :)

Comment (RachanaGJun20) [reply src/rachanaGJun20/String/StringReplaceFun.java:15]: > We should not write it like this. This is completely hardcoded... Your code wont work if name is maulik with same special character. and givenstr has no meaning then why to pass parameter if you want to say replace Kr with "" and so on...

yeah right got it.I have changed method calling , but not able to write other logic(without hardcoading) with out using loop. please suggest.

**Pull request merged**